packages/@aws-cdk/aws-events/lib/connection.ts

```diff
@@ -199,6 +199,7 @@ export abstract class HttpParameter {
         return {
           key: name,
           value,
+          isValueSecret: false,
```

Comment:
Why will this always be false? This PR needs context.

Reply:
Yes this value needs to be set to false to indicate the provided value is not secret. Since this is the render method from the `HttpParameter.fromString()` the value is always treated as not secret. If you like to provide a secret value you have to use the `HttpParameter.fromSecret()` method. 
The default value for `isValueSecret` seams to be `true` if not provided. See the updated PR description or linked issue.